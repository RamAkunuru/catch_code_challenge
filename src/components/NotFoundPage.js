import React, { Fragment } from 'react';
import { Link } from 'react-router-dom';

export default () => (
    <Fragment>
    404 - <Link to="/">GO home</Link>
</Fragment>
);
