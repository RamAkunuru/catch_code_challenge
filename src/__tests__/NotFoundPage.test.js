import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from "../components/NotFoundPage";

describe("NotFoundPage component", () => {
    test("render matches the snapshot", () => {
        const component = shallow(<NotFoundPage />);
        expect(component).toMatchSnapshot();
    });
});
