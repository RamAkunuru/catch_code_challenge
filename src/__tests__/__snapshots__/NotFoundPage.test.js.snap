// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component render matches the snapshot 1`] = `
<Fragment>
  404 - 
  <Link
    to="/"
  >
    GO home
  </Link>
</Fragment>
`;
